awscli/examples/healthlake/create-fhir-datastore.rst

```diff
@@ -17,4 +17,31 @@ Output::
         "DatastoreId": "(Datastore ID)"
     }
 
-For more information, see `Creating and monitoring a FHIR Data Store <https://docs.aws.amazon.com/healthlake/latest/devguide/working-with-FHIR-healthlake.html>`__ in the *Amazon HealthLake Developer Guide*.
+**Example 2: Create a SMART on FHIR-enabled HealthLake data store**
+
+    aws healthlake create-fhir-datastore \
+        --datastore-name "your-data-store-name" \
+        --datastore-type-version R4 \
+        --preload-data-config PreloadDataType="SYNTHEA" \
+        --sse-configuration '{ "KmsEncryptionConfig": {  "CmkType": "CUSTOMER_MANAGED_KMS_KEY", "KmsKeyId": "arn:aws:kms:us-east-1:your-account-id:key/your-key-id" } }' \
+        --identity-provider-configuration  file://identity_provider_configuration.json
+
+Contents of ``identity_provider_configuration.json``::
+
+    {
+      "AuthorizationStrategy": "SMART_ON_FHIR_V1",

```

Comment:
Indentation needs to be 4 spaces each. These are only six spaces and need to be increased to 8.